docs/protocol/substrate-cli.mdx

```diff
@@ -68,21 +68,21 @@ Creating a filesystem with a chunk (sunit) size of 32KB can prevent data fragmen
 
 **XFS**
 
-```
+```bash
```

Comment:
I'd suggest keeping the next three code blocks without `bash` instructions, as the result color scheme looks a bit weird. It highlights the words that wouldn't have been highlighted in a terminal.